app/frontends/jobs.ts

```diff
@@ -174,7 +174,7 @@ export async function getJobStatus(
         ({
           job,
           pagination,
-        } = await Job.byUsernameAndRequestId(tx, req.user, jobID, true, page, limit));
+        } = await Job.byUsernameAndRequestId(tx, req.user, jobID, false, page, limit));
```

Comment:
I misread this - isn't this going to make it so that links are not returned by the Job status endpoint? Maybe I'm missing some other change.

Reply:
I made a change to the function signature (swapping out the old boolean param for another). The old param was no longer needed because everything calling byUsernameAndRequestId currently requests links. I changed the boolean to signify whether you want stacOnly links, which for the job status code is false.

https://github.com/nasa/harmony/blob/1b6f5b9c9b3b3053c02904160ec4c06b868ee656/app/models/job-link.ts#L182

Reply:
Here is the change actually. https://github.com/nasa/harmony/pull/42/files#diff-cdf5c9e86f11ce00f32337e6562e6bbd9dfa25d732fa892573c8335bb04fa046L224-L225